examples/css-minifier/README.md

```diff
@@ -0,0 +1,23 @@
+# JavaScript Minifier Example
+
+A simple example of minifying javascript files before they're emitted using this plugin.
+
+## Trying out
+
+Install dependencies.
+
+```
+$ yarn
+```
+
+Run build.
+
+```
+$ yarn build
+```
+
+![Building](misc/screenshot.png)
+
+## Configuration
+
+Check the webpack configuration [here](webpack.config.js).
```

Comment:
Please update it as per your new example title, screenshot, etc. - everything here looks like a duplicate of previous example to me.

Reply:
i missed this one out. Will do